src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -365,8 +362,8 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 <!-- As we split up/refactor this file, put new imports here -->
 <Import Project="$(MSBuildThisFileDirectory)Xamarin.Android.D8.targets" Condition=" '$(AndroidDexTool)' == 'd8' " />
 <Import Project="$(MSBuildThisFileDirectory)Xamarin.Android.DX.targets" Condition=" '$(AndroidDexTool)' == 'dx' and '$(UsingAndroidNETSdk)' != 'true' " />
-<Import Project="$(MSBuildThisFileDirectory)Xamarin.Android.Aapt.targets" />
-<Import Project="$(MSBuildThisFileDirectory)Xamarin.Android.Aapt2.targets" />
+<Import Project="$(MSBuildThisFileDirectory)Xamarin.Android.Aapt.targets"  Condition=" '$(AndroidUseAapt2)' != 'true' and '$(UsingAndroidNETSdk)' != 'true' " />
+<Import Project="$(MSBuildThisFileDirectory)Xamarin.Android.Aapt2.targets" Condition=" '$(AndroidUseAapt2)' == 'true' or '$(UsingAndroidNETSdk)' == 'true' " />
```

Comment:
In a legacy project, if the `_ResolveAndroidTooling` later changes `$(AndroidUseAapt2)`, outputting a different value into `$(_AndroidUseAapt2)`, is it hypothetically possible users could get a build error because the matching AAPT or AAPT2 targets wouldn't be imported? (Maybe there's an existing automated test that covers this and ensures it isn't a concern.)

Reply:
Considering we ship aapt2 now, @dellis1972 do you think I should turn these into build _errors_ now?

https://github.com/xamarin/xamarin-android/blob/9bc45b58ce0641890a0a3fbd6498eac515e9555a/src/Xamarin.Android.Build.Tasks/Tasks/ResolveAndroidTooling.cs#L141-L154

I don't think we need to emit a warning and fall back to aapt(1) anymore, we should just give a nice error message instead? This would happen if `aapt2` didn't exist on disk, or didn't return a valid version number.

Reply:
I went ahead and made these changes in a second commit. It seems like it makes us able to clean up a lot of things?

I also updated some more error messages that @brendanzagaeski might want to take a look at. XA0111 and XA0112 should be pretty rare if we redistribute aapt2, but maybe they need more instructions?